hat/test/org/hatdex/hat/api/service/applications/ApplicationsServiceSpec.scala

```diff
@@ -261,7 +262,39 @@ class ApplicationsServiceSpec(implicit ee: ExecutionEnv) extends PlaySpecificati
         setup ← service.setup(HatApplication(notablesAppMissing, true, true, true, None, None, None))
       } yield setup
 
-      result must throwA[RuntimeException].await(1, 20.seconds)
+      result must throwA[HatApplicationSetupException].await(1, 20.seconds)
+    }
+  }
+
+  "Application `setup` method for applications with dependencies" should {
+    "Enable plug dependencies" in {
+      val service = application.injector.instanceOf[ApplicationsService]
+      val result = for {
+        app <- service.applicationStatus(notablesAppDebitlessWithPlugDependency.id)
+        setup <- service.setup(app.get)
```

Comment:
`get`

Reply:
I deem it acceptable in the tests. The only way it will throw None.get is if the test is setup wrongly and I see now way of gracefully recovering the wrong test :)